terraform/instances.tf

```diff
@@ -6,6 +6,18 @@ provider "aws" {
   profile = "habitat"
 }
 
+locals {
+  # We have a few instances that run on Linux, and all should have the
+  # same SystemD unit file. We declare it here to keep things DRY.
+  hab_sup_systemd_unit_content = templatefile(
```

Comment:
Should we follow the same pattern for the non-Linux systemds as well by adding them all to local?

Reply:
I was initially thinking "no", since this one is the only one that is being used in multiple contexts.

However, bringing them all together in the same physical space could make it easier to spot commonalities / differences between them.

I think you could make a decent case either way; I'm happy to do it in whichever way you prefer.

Reply:
Since you're already in the code, I'd say let's do it.. it will reduce the context switching in the future.. these are all great improvements you're bringing in!

Reply:
Addressed in https://github.com/habitat-sh/builder/pull/1252/commits/cb7ecef7b542ff6d5cc98359b6ecd0307e89f283